packages/transaction-controller/src/helpers/EtherscanRemoteTransactionSource.ts

```diff
@@ -33,6 +34,10 @@ export class EtherscanRemoteTransactionSource
 
   #isTokenRequestPending: boolean;
 
+  #mutex = new Mutex();
+
+  ETHERSCAN_RATE_LIMIT_INTERVAL = 5000;
```

Comment:
Minor, but can this be a standard `const` in this file rather than a property on each instance?

Reply:
I'll handle this one

Reply:
Addressed [here](https://github.com/MetaMask/core/pull/3875)